packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -380,6 +380,27 @@ contract EpochRewards is Ownable, Initializable, UsingPrecompiles, UsingRegistry
     _updateTargetVotingYield();
   }
 
+  /**
+   * @notice Determines if the reserve is low enough to demand a diversion from
+   *    the community reward. Targets initial ratio of 2 with a linear decline
+   *    until 25 years have passed where the target ratio will be 1.
+   */
+  function isReserveLow() external view returns (bool) {
+    // Target reserve ratio = 2 - time in second / 25 years
```

Comment:
Nitty: Rethinking this, I find "target" a bit misleading as a term here - it is more a critical value. How about renaming this to criticalRatio?